frappe/website/doctype/web_form/templates/web_form.html

```diff
@@ -19,7 +19,7 @@ <h1>{{ _(title) }}</h1>
 		<a href="{{ cancel_url or pathname }}" class="btn btn-default btn-sm">
 	    	{{ _("Cancel") }}</a>
 		{%- endif %}
-		{% if not is_list %}
+		{% if not is_list and layout|length == 1 %}
```

Comment:
This modification disables this button for multi-page forms or we will have to start using javascript for status management.

What is is_list used for?

Reply:
Why `and layout|length == 1` ?

is_list is used to check whether the webform is in the list view or in the form view.